src/mirror.c

```diff
@@ -200,11 +200,34 @@ static enum swupd_code set_mirror_url(char *url)
 	return ret;
 }
 
+static bool mirror_is_set(void)
+{
+	bool mirror_set;
+	char *version_path;
+	char *content_path;
+
+	version_path = mk_full_filename(path_prefix, MIRROR_VERSION_URL_PATH);
+	content_path = mk_full_filename(path_prefix, MIRROR_CONTENT_URL_PATH);
```

Comment:
Content url isn't used by handle_mirror_if_stale, so it doesn't make sense to test mirror if version_path isn't set.

Reply:
`handle_mirror_if_stale` only uses the version_url to see if the mirror is stale, but when a mirror is set, both version_url and content_url are set, so to actually claim that a valid mirror is set both of them would need to be set, wouldn't them? That is why I included both in the `mirror_is_set` function. Do you really think I should get rid of the content_url check in this function?